### DISCUSSION TITLE
【求助】我的頁面一直要求我更新繩網小助手

### DISCUSSION THREAD

**Discussion by BY-performer** [share121/inter-knot#2539] [崩坏：星穹铁道]
一直顯示「请更新“绳网小助手”，最新版本为 1.4.0」，但是我已經更新過了，而且顯示的最新版本是1.14.0 ... ...
我還試著重新安裝了好幾次...
有大佬知道是怎麼回事嗎...
![image](https://github.com/user-attachments/assets/fcf2b0ed-fba6-4550-bf62-2c3c51e19a89)
![image](https://github.com/user-attachments/assets/c2ebe365-182a-4f32-ab36-009a0b1b5d87)
![image](https://github.com/user-attachments/assets/26013ee0-8376-41f8-89b7-36b27e724f52)

**Comment by HifumiMizuhara** [comment]
尝试清空Cache。

**Comment by BY-performer** [comment]
有成功了!謝謝各位大佬!
